### DISCUSSION TITLE
Right align label for input-group?

### DISCUSSION THREAD

**Discussion by Robert-M-Muench** [saadeghi/daisyui#406] [Q&A] [answered]
How can I right align the labels?

See: https://play.tailwindcss.com/VD4dLarrG2

**Comment by saadeghi** [comment, marked as answer]
Change the positions: https://play.tailwindcss.com/uaJ0u9AwjE

**Reply by Robert-M-Muench** [reply]
I want the label on the left side, but the text inside the label right aligned.

**Reply by Robert-M-Muench** [reply]
Ok, got it. I have to use `justify-end` for the SPAN.

**Reply by saadeghi** [reply]
Yeah that works too.
